core/src/test/java/org/apache/carbondata/core/metadata/converter/ThriftWrapperSchemaConverterImplTest.java

```diff
@@ -1729,7 +1729,7 @@
     TableInfo actualResult = thriftWrapperSchemaConverter
         .fromExternalToWrapperTableInfo(externalTableInfo, "dbName", "tableName", "/path");
     assertEquals(time, actualResult.getLastUpdatedTime());
-    assertEquals("dbName_tableName", actualResult.getTableUniqueName());
+    assertEquals("dbname_tablename", actualResult.getTableUniqueName());
```

Comment:
why need to change it?

Reply:
After removing the metacache, all the carbontables had table name in lowercase